src/main/java/org/jabref/logic/importer/OpenDatabase.java

```diff
@@ -29,31 +31,26 @@ private OpenDatabase() {
      *
      * @param name Name of the BIB-file to open
      * @return ParserResult which never is null
+     * @deprecated use {@link #loadDatabase(Path, ImportFormatPreferences, FileUpdateMonitor)} instead
      */
+    @Deprecated
     public static ParserResult loadDatabase(String name, ImportFormatPreferences importFormatPreferences, FileUpdateMonitor fileMonitor) {
-        File file = new File(name);
-        LOGGER.info("Opening: " + name);
+        LOGGER.debug("Opening: " + name);
```

Comment:
Doesn't "Opening {}", name work? - I think, this is the style one does in Java, because otherwise the debug string is always calculated even if the statement is not set.

Reply:
Isn't that micro optimization?